README.markdown

```diff
@@ -189,7 +189,7 @@ github "PromiseKit/MapKit" ~> 1.0
 
 ## Choose Your Networking Library
 
-`NSURLSession` is typically inadequate; choose from [Alamofire] or [OMGHTTPURLRQ]:
+`URLSession` is typically inadequate; choose from [Alamofire] or [OMGHTTPURLRQ]:
```

Comment:
`NSURLSession` is the class name IMO. When you talk about this class in person, what do you say? Everyone I know still says "NSURLSession" and I expect them to for quite some time to come.